templates/controller-service.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
+{{- if .Values.controller.ingress.enabled }}
+  type: ClusterIP
```

Comment:
should this be configurable?

Reply:
I don't think so. The ingress is pointing to it and that's how it's exposed. Using NodePort this gets also exposed in a different way and using LoadBalancer exposes it also in a different way I think, but it sounds very confusing. ClusterIP exposes the service only on a cluster internal ip and the ingress does the external access bit. Unless I am missing something... :-)

Reply:
sure, it's a thing that is sometimes made configurable to make it more flexible in other environments but let's see what the neuvector people say